cmd/scanner-trivy/main.go

```diff
@@ -1,16 +1,19 @@
 package main
 
 import (
+	"os"
+	"os/signal"
+	"syscall"
+
 	"github.com/aquasecurity/harbor-scanner-trivy/pkg/etc"
+	"github.com/aquasecurity/harbor-scanner-trivy/pkg/ext"
 	"github.com/aquasecurity/harbor-scanner-trivy/pkg/http/api"
-	"github.com/aquasecurity/harbor-scanner-trivy/pkg/http/api/v1"
+	v1 "github.com/aquasecurity/harbor-scanner-trivy/pkg/http/api/v1"
```

Comment:
We don't but this is my IDE (Goland) being overly clever about it. I can't even remove it manually because the file watcher will auto add it. 
<img width="180" alt="image" src="https://user-images.githubusercontent.com/1254783/77127145-af352600-6a08-11ea-8020-9f73462fa662.png">